adobe-esign-connector/src/com/axonivy/connector/adobe/esign/connector/auth/oauth/VarTokenStore.java

```diff
@@ -0,0 +1,49 @@
+package com.axonivy.connector.adobe.esign.connector.auth.oauth;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+import ch.ivyteam.ivy.environment.Ivy;
+
+/**
+ * Class for storing OAuth token to Variable
+ * @author stefan.masek
+ *
+ */
+public class VarTokenStore {
+	private final String varName;
+
+	public static VarTokenStore get(String varName) {
+		return new VarTokenStore(varName);
+	}
+
+	VarTokenStore(String varName) {
+		this.varName = varName;
+	}
+
+	/**
+	 * Loads Variable and tries to create the token object {@link Token}
+	 * @return
+	 */
+	public Token getToken() {
+		String tokenVar = Ivy.var().get(varName);
+		Gson gson = new Gson();
```

Comment:
Create a singleton for Jackson mapper and use it instead of Gson. Ex JacksonUtils

Reply:
Use JacksonUtils instead